core/src/main/java/com/google/cloud/sql/core/InternalConnectorRegistry.java

```diff
@@ -114,14 +119,24 @@ public static synchronized InternalConnectorRegistry getInstance() {
    * Calls shutdown on the singleton and removes the singleton. After calling shutdownInstance(),
    * the next call to getInstance() will start a new singleton instance.
    */
-  public static synchronized void shutdownInstance() {
+  public static synchronized void resetInstance() {
```

Comment:
This should be private right?

Reply:
It should be public so that `ConnectorRegistry.shutdown()` can call it. `ConnectorRegistry` is in a different package.